Dashboard/src/routes-nav/Navigation.js

```diff
@@ -1,51 +1,94 @@
-import React, { useState } from "react"
+import React from "react"
+import { useHistory } from "react-router-dom"
 import {
-  Collapse,
-  Navbar,
-  NavbarToggler,
-  NavbarBrand,
-  Nav,
-  NavItem,
-  NavLink,
-} from "reactstrap"
+  BottomNavigation,
+  BottomNavigationAction,
+  makeStyles,
+} from "@material-ui/core"
+import Computer from "@material-ui/icons/Computer"
```

Comment:
import { Computer as Computer, CreditCard as CreditCard, etc.} from '@material-ui/icons' <- see if this works. If not lets introduce a babel config edit towards the bottom of the page here. https://material-ui.com/guides/minimizing-bundle-size/